manim/mobject/logo.py

```diff
@@ -71,13 +67,15 @@ def __init__(self, dark_theme: bool = True):
         self.circle = Circle(color=logo_green, fill_opacity=1).shift(LEFT)
         self.square = Square(color=logo_blue, fill_opacity=1).shift(UP)
         self.triangle = Triangle(color=logo_red, fill_opacity=1).shift(RIGHT)
-        self.add(self.triangle, self.square, self.circle, self.M)
+        self.shape = VGroup(self.triangle, self.square, self.circle)
```

Comment:
Just a question of semantics: should `self.shape` be called `self.shapes`?

Reply:
Yeah, thought about that as well. `shapes` would be nicer imo.

Reply:
I'd also be in favor of refactoring. Could you change the identifier name, or should I add a commit?